Workflow/Modules/ModuleBase.py

```diff
@@ -357,7 +357,7 @@ def setApplicationStatus( self, status, sendFlag = True ):
       # The application status won't be updated in case the workflow or the step is failed already
       if not isinstance( status, basestring ):
         status = str( status )
-      self.log.verbose( 'setJobApplicationStatus(%d, %s)' % ( self.jobID, status ) )
+      self.log.verbose( 'setJobApplicationStatus(%s, %s)' % ( self.jobID, status ) )
```

Comment:
No, it's not! `self.jobID` is an `int`, not a `string`. And in this PR, for this line only, you are treating it like a string.